tests/unit/PolicyUtilsTest.ts

```diff
@@ -1,13 +1,146 @@
+        role: 'admin',
+        submitsTo: '',
+    },
+    'admin@test.com': {
+        email: 'admin@test.com',
+        role: 'admin',
+        submitsTo: '',
+    },
+    'employee@test.com': {
+        email: 'employee@test.com',
+        role: 'user',
+        submitsTo: 'admin@test.com',
+    },
+    'categoryapprover1@test.com': {
+        email: 'categoryapprover1@test.com',
+        role: 'user',
+        submitsTo: 'admin@test.com',
+    },
+    'categoryapprover2@test.com': {
+        email: 'categoryapprover2@test.com',
+        role: 'user',
+        submitsTo: 'admin@test.com',
+    },
+    'tagapprover1@test.com': {
+        email: 'tagapprover1@test.com',
+        role: 'user',
+        submitsTo: 'admin@test.com',
+    },
+    'tagapprover2@test.com': {
+        email: 'tagapprover2@test.com',
+        role: 'user',
+        submitsTo: 'admin@test.com',
+    },
+};
+
+const personalDetails: PersonalDetailsList = {
+    '1': {
+        accountID: 1,
+        login: 'admin@test.com',
+    },
+    '2': {
+        accountID: 2,
+        login: 'employee@test.com',
+    },
+    '3': {
+        accountID: 3,
+        login: 'categoryapprover1@test.com',
+    },
+    '4': {
+        accountID: 4,
+        login: 'categoryapprover2@test.com',
+    },
+    '5': {
+        accountID: 5,
+        login: 'tagapprover1@test.com',
+    },
+    '6': {
+        accountID: 6,
+        login: 'tagapprover2@test.com',
+    },
+    '7': {
+        accountID: 7,
+        login: 'owner@test.com',
+    },
+};
+
+const rules = {
+    approvalRules: [
+        {
+            applyWhen: [
+                {
+                    condition: 'matches',
+                    field: 'category',
+                    value: 'cat1',
+                },
+            ],
+            approver: 'categoryapprover1@test.com',
+            id: '1',
+        },
+        {
+            applyWhen: [
+                {
+                    condition: 'matches',
+                    field: 'tag',
+                    value: 'tag1',
+                },
+            ],
+            approver: 'tagapprover1@test.com',
+            id: '2',
+        },
+        {
+            applyWhen: [
+                {
+                    condition: 'matches',
+                    field: 'category',
+                    value: 'cat2',
+                },
+            ],
+            approver: 'categoryapprover2@test.com',
+            id: '3',
+        },
+        {
+            applyWhen: [
+                {
+                    condition: 'matches',
+                    field: 'tag',
+                    value: 'tag2',
+                },
+            ],
+            approver: 'tagapprover2@test.com',
+            id: '4',
+        },
+    ],
+};
+
+const ownerAccountID = 7;
+const adminAccountID = 1;
+const employeeAccountID = 2;
+const categoryapprover1AccountID = 3;
+const categoryapprover2AccountID = 4;
+const tagapprover1AccountID = 5;
+const tagapprover2AccountID = 6;
```

Comment:
Could be nice to define these above `PersonalDetailsList` and use them when defining `PersonalDetailsList`?